kong/utils_test.go

```diff
@@ -4209,6 +4209,12 @@ func Test_FillPluginsDefaultsWithPartials(t *testing.T) {
 			errString: "no 'config' field found in schema",
 		},
 	}
+	// Kong Enterprise added `throttling` field and assigned default values since 3.11.
+	// We ignore the field here to be compatible with 3.10.
+	// TODO: set default value in the expected plugin configuration when Kong version >= 3.11?
```

Comment:
I believe we should set this default in the expected plugin configuration for 3.11+ and not ignore it in the test assertion below. WDYT?